fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAORequestFactory.java

```diff
@@ -0,0 +1,31 @@
+package com.github.dedis.popstellar.repository.remote;
+
+import androidx.annotation.NonNull;
+
+import com.tinder.scarlet.websocket.okhttp.request.RequestFactory;
+
+import okhttp3.Request;
+
+public class LAORequestFactory implements RequestFactory {
+
+  @NonNull private static String url;
```

Comment:
The static url is annoted @NonNull but it is not initialized before an instance is created.